locales/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-gettext_templates(${CMAKE_CURRENT_SOURCE_DIR} ${ALL_SOURCES})
-gettext_compile(${CMAKE_CURRENT_SOURCE_DIR} share/locale)
-SET_DIRECTORY_PROPERTIES(PROPERTIES CLEAN_NO_CUSTOM TRUE)
-export_var(GETTEXT_ENABLED)
+if (NOT WIN32)
+    gettext_templates(${CMAKE_CURRENT_SOURCE_DIR} ${ALL_SOURCES})
```

Comment:
Note that the build breakage is not actually caused by gettext itself. I believe the issue is a Windows command line length limit. With a huge number of files on the command line to xgettext, some number of arguments are truncated. We may need to change how xgettext is called for Windows. For projects with smaller number of source files (e.g. cpp-hocon and even leatherman), xgettext works just fine on Windows.